tests/unit/src/utils.spec.ts

```diff
@@ -110,6 +110,26 @@ describe('utils', function () {
 
       expect(npmConfig).toHaveProperty('registry', 'http://my-private-registry.com');
     });
+
+    describe.each([
+      { value: undefined, expected: true },
+      { value: null, expected: true },
+      { value: '', expected: true },
+      { value: 'true', expected: true },
+      { value: true, expected: true },
+      { value: 'false', expected: false },
```

Comment:
Not sure if we want to be interpolating strings here like this. Open to changing this.